sdk/src/types/block/capabilities.rs

```diff
@@ -0,0 +1,207 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use alloc::boxed::Box;
+use core::marker::PhantomData;
+
+use derive_more::Deref;
+use packable::{error::UnpackErrorExt, prefix::BoxedSlicePrefix, Packable};
+
+/// A list of bitflags that represent capabilities.
+#[derive(Debug, Deref)]
+#[repr(transparent)]
+pub struct Capabilities<Flag> {
+    #[deref]
+    bytes: BoxedSlicePrefix<u8, u8>,
+    _flag: PhantomData<Flag>,
+}
+
+impl<Flag> Capabilities<Flag> {
+    pub(crate) fn from_bytes(bytes: BoxedSlicePrefix<u8, u8>) -> Self {
+        Self {
+            bytes,
+            _flag: PhantomData,
+        }
+    }
+
+    /// Returns a [`Capabilities`] with every possible flag disabled.
+    pub fn none() -> Self {
+        Self::default()
+    }
+
+    /// Returns whether every possible [`AddressCapabilityFlag`] is disabled.
```

Comment:
There is still a bunch of `AddressCapabilityFlag` in this file

Reply:
oops